src/modelAnalysis/rumba/rumba.m

```diff
@@ -0,0 +1,294 @@
+function [RUMBA_outputs, UpRegulated, DownRegulated, MetConnectivity1, MetConnectivity2] = rumba(model1, model2, completeModel, sampling, maxMetConn, RxnsOfInterest, GenesOfInterest, NormalizePointsOption, PValCuttoff, MaxNumPoints, LoopRxnsToIgnore, verboseTag)
+% RUMBA predicts which reactions significantly change their

```

Comment:
There seem to be a lot of optional parameters here. (checks for nargin <4).
A: These should be marked as optional inputs (and their defaults mentioned)
B: For more than 3 optional parameters I strongly suggest using a inputParser and allowing parameter/value pairs. It's otherwise very easy to get the wrong input at the wrong position without noticing it, if the input values are similar.